src/components/SignIn/index.tsx

```diff
@@ -36,6 +38,11 @@ export interface SignInProps {
     handleChangeFocusField: (value: string) => void;
     changePassword: (value: string) => void;
     changeEmail: (value: string) => void;
+    captchaType?: 'recaptcha' | 'geetest' | 'none';
+    renderCaptcha?: JSX.Element | null;
+    reCaptchaSuccess?: boolean;
+    geetestCaptchaSuccess?: boolean;
```

Comment:
can we combine reCaptchaSuccess and geetestCaptchaSuccess? (we can have only one type of captcha at the same time)